performance_manager/lib/gtfs_static_table.py

```diff
@@ -232,6 +232,11 @@ def transform_data_tables(static_tables: List[StaticTableDetails]) -> None:
                     .astype("Int64")
                 )
 
+        table.data_table = table.data_table.fillna(numpy.nan).replace(
```

Comment:
whats this about?

Reply:
Dataframes can have NULL values represented in a few slightly different ways, but the only one compatible with writing to the RDS with sqlalchemy is the Python `None` dtype. 

So this line turns any na/NULL values into a python `None` in the dataframe:
`table.data_table = table.data_table.fillna(numpy.nan).replace(
            [numpy.nan], [None]
        )`

Then with the GTFS static data I also found an issue where values that are NULL / blank are loaded into the dataframe as an empty string "". This was breaking some queries where we filter on NULL in the RDS.

So this line replaces any empty strings with Python `None` dtypes, that will turn into NULL in the RDS:
`table.data_table = table.data_table.replace([""], [None])`